fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java

```diff
@@ -61,23 +71,19 @@
 import org.springframework.stereotype.Service;
 
 @Service
+@AllArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
RequiredArgsConstructor makes more sense here plus there's no need to put the Autowired annotation onto the generated constructor since Spring 4.3+ supports constructor dependency injection without it.